sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ResolveDefaultColumnsUtil.scala

```diff
@@ -344,3 +358,13 @@ object ResolveDefaultColumns {
     }
   }
 }
+
+/**
+ * Represents a table with an underlying raw schema to use for resolving DEFAULT column references.
+ * The 'ignoreColumnsForInserts' field represents a set of column names to exclude from
```

Comment:
Let's move(or duplicate) the comment to the method

Reply:
Good idea, done.